Specs/DataSources/ModelVisualizerSpec.js

```diff
@@ -396,12 +399,11 @@ describe(
         return state !== BoundingSphereState.PENDING;
       }).then(function () {
         expect(state).toBe(BoundingSphereState.DONE);
-        expect(modelPrimitive._clampedModelMatrix).toBeDefined();
         const expected = BoundingSphere.clone(
           modelPrimitive.boundingSphere,
           new BoundingSphere()
         );
-        expect(result).toEqual(expected);
+        expect(result).not.toEqual(expected);
```

Comment:
Can we be more specific on what's expected here?

Reply:
Added a more detailed check that uses the value returned from `sampleTerrainMostDetailed`.